package.json

```diff
@@ -2307,11 +2306,13 @@
         "node-gyp-build": "^4.2.3",
         "node-has-native-dependencies": "^1.0.2",
         "node-html-parser": "^1.1.13",
+        "node-loader": "^2.0.0",
```

Comment:
Is this used? I don't see it added in the webpack files.

Reply:
Oh I don't need it. Let me remove.